cli/src/cli.mjs

```diff
@@ -400,29 +402,48 @@ if ( parsed.help ) {
     console.log(ex);
   }
 
-} else if ( parsed.session ) {
-  const record = parsed.session;
-  if ( record === 'start' ) {
+} else if ( parsed.session || parsed.sessionOutput || parsed.sessionOutputPath) {
+  if ( parsed.session && parsed.session == 'start' ) {
     msg(`Starting session...`);
-    axios.post(url(host, port, '/api/v1/session/start'), undefined)
-    .then(function (response) {
+    try {
+      let response = await axios.post(url(host, port, '/api/v1/session/start'), undefined);
       console.log(response.data);
```

Comment:
It might make sense to create a ticket in the backlog to find these kinds of "naked" log statements and change them so they log a bit more info / context (so if you look at the log it is very clear what it means / where it came from in the code).